src/app/Modal/DeleteWarningModal.tsx

```diff
@@ -46,62 +46,58 @@ export interface DeleteWarningProps {
   visible: boolean;
   onAccept: () => void;
   onClose: () => void;
-  setShowDialog: () => void;
-  checkbox?: boolean;
-  setCheckbox?: React.Dispatch<React.SetStateAction<boolean>>;
+  disableDialog: () => void;
 }
 
-export const DeleteWarningModal = ({ warningType , items, visible, onAccept, onClose, setShowDialog, checkbox, setCheckbox}: DeleteWarningProps): JSX.Element => {
+export const DeleteWarningModal = ({ warningType , items, visible, onAccept, onClose, disableDialog}: DeleteWarningProps): JSX.Element => {
   const [doNotAsk, setDoNotAsk] = useState(false);
   const realWarningType = getFromWarningMap(warningType);
 
-  const description = `${realWarningType?.description}${(typeof items === 'undefined' || items.length <= 1) ?  "":"s"}: [${items?.join(", ")}]. ${realWarningType?.descriptionPartTwo}`
+  const description = 
+    <TextContent>
+      <Text>
+        {realWarningType?.description}{(typeof items === 'undefined' || items.length <= 1) ?  "":"s"}: [{items?.join(", ")}]
+        <br />       
+        {realWarningType?.descriptionPartTwo}
+      </Text>
+    </TextContent>
 
   const onAcceptClose = () => {
     onAccept();
     onClose();
     if (doNotAsk) {
-      setShowDialog();
+      disableDialog();
     }
   }
   
   return (
-      <Modal
-        title={`Permanently ${realWarningType?.title}?`}
-        description={description}
-        aria-label={realWarningType?.ariaLabel}
-        titleIconVariant="warning"
-        variant={ModalVariant.small}
-        isOpen={visible}
-        showClose
-        onClose={onClose}
-        actions={[  
-          <Stack hasGutter key="modal-footer-stack">
-            <Split key="modal-footer-split">
-              <Button variant="danger" onClick={() => onAcceptClose()}>
-                Delete
-              </Button>
-              <Button variant="link" onClick={onClose}>
-                Cancel
-              </Button>
-            </Split>
-          </Stack> 
-        ]}
-      >
-        <Stack hasGutter key="modal-checkboxes-stack">
-          {(realWarningType === DeleteAutomatedRules) && (typeof setCheckbox !== 'undefined') &&
-          <Checkbox id="clean-rule-enabled" 
-            label="Clean" 
-            description={`Clean will stop any Active Recordings that ${items![0]} created.`}
-            isChecked={checkbox} 
-            onChange={(checked) => setCheckbox(checked)}
-          />}
-          <Checkbox id="do-not-ask-enabled" 
-            label="Don't ask me again" 
-            isChecked={doNotAsk} 
-            onChange={(checked) => setDoNotAsk(checked)}
-          />
-        </Stack>
-      </Modal>  
+    <Modal
+      title={`Permanently ${realWarningType?.title}?`}
```

Comment:
I think it'd be better to move the `Permanently ` prefix into the actual "warning type title". This keeps the component more generic, and easier to localize in the future.

Reply:
![image](https://user-images.githubusercontent.com/69827283/176738226-0e554a9a-0eab-46d2-b5aa-2d07e34ed0b7.png)
I was previously mapping the title to be the label text that's shown on the setting, should I use the id instead like this, or would you rather me just add a new field in the DeletionWarning object called `label` or something?

Reply:
Hmm, that's fair. Just using the `id` works well enough for now, but again that'd be a deficiency later on when we get around to localizing this thing and need that to display in other languages.

I guess a `label` and a `title` property would work, where ex. `label: Delete Automated Rule` and `title: Permanently delete Automated Rule`. This looks a little redundant but it's also clear and unambiguous, and we can very cleanly extract those strings and localize them.